edx_proctoring/views.py

```diff
@@ -779,3 +780,30 @@ def get(self, request):
             user_id=request.data.get('user_id', None),
             course_id=request.data.get('course_id', None)
         ))
+
+
+class ProctoredExamAttemptReviewStatus(AuthenticatedAPIView):
+    """
+    Endpoint for the Active Exams for a user.
+    edx_proctoring/v1/proctored_exam/attempt/(<attempt_id>)/review_status$
+
+    Supports:
+        HTTP PUT: Update the is_status_acknowledge flag
+    """
+    def put(self, request, attempt_id):     # pylint: disable=unused-argument
+        """
+        Update the is_status_acknowledge flag for the specific attempt
+        """
+        try:
+            update_exam_attempt(attempt_id, is_status_acknowledged=True)
+
+            return Response(
+                status=status.HTTP_200_OK
+            )
+
+        except ProctoredBaseException, ex:
```

Comment:
`update_exam_attempt` sends `StudentExamAttemptDoesNotExistsException`. change the exception to `StudentExamAttemptDoesNotExistsException `